airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -475,7 +475,7 @@ public void testScheduledSync() throws Exception {
     // todo: wait for two attempts in the UI
     // if the wait isn't long enough, failures say "Connection refused" because the assert kills the
     // syncs in progress
-    sleep(Duration.ofMinutes(2).toMillis());
+    sleep(Duration.ofMinutes(4).toMillis());
```

Comment:
This is taking slightly longer for Kube. What was happening is we were checking this in the middle of an actual run.

Should definitely change this to properly query the api. Will do so in a separate PR.